tests/shared/README.md

```diff
@@ -0,0 +1,4 @@
+This folder contains test code that is meant to be reused in more than one backend-specific (or agnostic) test subfolder, such as reusable fixtures, parsers, comparators etc.
```

Comment:
What do you think about renaming `tests/shared` -> `tests/utils` or `tests/test_utils`?

Reply:
There's always an option to introduce `utils` into a name of the file or a foler at a lower level. At this level I need to make evident that this is the code that is designed to be "shared".